### ISSUE TITLE
Some BytesTest fails

### ISSUE THREAD

**Issue by peter-lawrey** [OpenHFT/Chronicle-Bytes#43]

**Comment by RobAustin** [comment]
The failing test are 

```
net.openhft.chronicle.bytes.BytesTest#testBigDecimalText
net.openhft.chronicle.bytes.MappedFileTest#testReferenceCounts
net.openhft.chronicle.bytes.ByteStoreTest#testStream2
net.openhft.chronicle.bytes.algo.OptimisedBytesStoreHashTest#testApplyAsLong
net.openhft.chronicle.bytes.BytesTest#testParseUtf8High
```

**Comment by dpisklov** [comment]
Created #46 and #47 to fix bugs uncovered by net.openhft.chronicle.bytes.MappedFileTest#testReferenceCounts and net.openhft.chronicle.bytes.algo.OptimisedBytesStoreHashTest#testApplyAsLong

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.101](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
